docs/services/azure-services-guide.md

```diff
@@ -262,124 +209,35 @@ You are free to create more Azure `SQLServerClass` instances to define more
-
-```yaml
-cat > app-project1-dev-namespace.yaml <<EOF
----
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: app-project1-dev
-EOF
-
-kubectl apply -f app-project1-dev-namespace.yaml
-```
-
-* You should see the following output:
-
-> namespace/app-project1-dev created
-
-#### Portable Resource Classes
-
-Portable resource classes are used to define a class of service in a single
-namespace for an abstract service type. We want to define our Azure
-`SQLServerClass` as the standard MySQL class of service in the namespace that
-our Wordpress resources will live in.
-
-* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace
-  `app-project1-dev` and create it:
-
-```yaml
-cat > mysql-standard.yaml <<EOF
----
-apiVersion: database.crossplane.io/v1alpha1
-kind: MySQLInstanceClass
-metadata:
-  name: mysql-standard
-  namespace: app-project1-dev
-classRef:
-  kind: SQLServerClass
-  apiVersion: database.azure.crossplane.io/v1alpha2
-  name: azure-mysql-standard
-  namespace: azure-infra-dev
-EOF
-
-kubectl apply -f mysql-standard.yaml
-```
-
-* You should see the following output:
-
-> mysqlinstanceclass.database.crossplane.io/mysql-standard created
-
-* You can verify creation with the following command and output:
-
-```bash
-$ kubectl get mysqlinstanceclasses -n app-project1-dev
-NAME             AGE
-mysql-standard   27s
-```
-
-Once again, you are free to create more `MySQLInstanceClass` instances in this
-namespace to define more classes of service. For instance, if you created
-`mysql-azure-large` above, you may want to create a `MySQLInstanceClass` named
-`mysql-large` that references it. You may also choose to create MySQL resource
-classes for other non-Azure providers, and reference them for a class of service
-in the `app-project1-dev` namespace.
-
-You may specify *one* instance of a portable class kind as *default* in each
-namespace. This means that the portable resource class instance will be applied
-to claims that do not directly reference a portable class. If we wanted to make
-our `mysql-standard` instance the default `MySQLInstanceClass` for namespace
-`app-project1-dev`, we could do so by adding a label:
-
-```yaml
----
-apiVersion: database.crossplane.io/v1alpha1
-kind: MySQLInstanceClass
-metadata:
-  name: mysql-standard
-  namespace: app-project1-dev
-  labels:
-    default: "true"
-classRef:
-  kind: SQLServerClass
-  apiVersion: database.azure.crossplane.io/v1alpha2
-  name: azure-mysql-standard
-  namespace: azure-infra-dev
-```
-
 #### Resource Claims
 
-Resource claims are used to create external resources by referencing a class of
-service in the claim's namespace. When a claim is created, Crossplane uses the
-referenced portable class to find a cloud-specific resource class to use as the
-configuration for the external resource. We need a to create a claim to
-provision the MySQL database we will use with Azure.
+Resource claims are used to create external resources by being scheduled to a
```

Comment:
Like showing the options. Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.

Reply:
Went with a link to the `concepts.md` section on dynamic and static provisioning. Happy to update if you strongly prefer link to quickstart.